case-study/sqls/issue_pr-resolve_til1stresponse-average_period/sql

```diff
@@ -0,0 +1,62 @@
+SELECT repo_id,
+    repo_name,
```

Comment:
You have a group by `repo_id` here so you need to add an aggregation function to `repo_name`, `anyHeavy` will do.

Reply:
I've added `anyHeavy` function in line 23. Will it cause a difference? (I am not very sure where to put it in.)

Reply:
Nope, you need to add aggregation function for every aggregation like group by, `anyHeavy` will work too.

Reply:
Do you mean that `GROUP BY repo_id` will gather repo names, which is the second overlap situation of different `repo_names` with a same repo_id, and there should be an `anyHeavy` function here as well as line 23?

Reply:
Yes, in line 23, the `anyHeavy` will select most used repo_name for certain issue because you group by repo_id and issue_number in that SQL. So here you need another `anyHeavy` to select the most used repo_name for all the issues in this repo, the repo_name may have more than one value.

Reply:
Thank you! I get it!

Reply:
No problem~